build.sbt

```diff
@@ -2,18 +2,12 @@ import play.core.PlayVersion
 import sbt.Keys._
 import sbt._
 import uk.gov.hmrc.DefaultBuildSettings._
-import uk.gov.hmrc.{SbtAutoBuildPlugin, _}
 import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin
 import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin._
 import uk.gov.hmrc.versioning.SbtGitVersioning
+import uk.gov.hmrc.{SbtAutoBuildPlugin, _}
```

Comment:
All the selenium scripts and code are copied from front-end  -  the only new bits are `pages` and `features` packages

Reply:
Also the `/run_selenium_system_test_local_chrome_service_manager.sh` script is not copied from frontend as the stride stubs are not running via service manager